src/cr/cube/cubepart.py

```diff
@@ -1394,6 +1460,34 @@ def min_base_size_mask(self):
     def name(self):
         return self.rows_dimension_name
 
+    def pairwise_significance_means_p_vals(self, column_idx):
```

Comment:
I can't see from diff, is this on `_Stripe`? If so, what does the `column_idx` mean (i.e. do we even need it on a stipe)?

Reply:
this has to be deleted, and btw it exists in the strand ... but nevermind this is not useful ... i;ll get rid of it